src/Event/ConstructEvent.php

```diff
@@ -8,21 +7,23 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
 namespace WyriHaximus\TwigView\Event;
 
 use Cake\Event\Event;
+use Twig\Environment;
 use WyriHaximus\TwigView\View\TwigView;
 
 class ConstructEvent extends Event
 {
     const EVENT = 'TwigView.TwigView.construct';
 
     /**
-     * @param TwigView $twigView
-     * @param \Twig_Environment $twig
+     * @param  TwigView          $twigView
+     * @param  \Twig\Environment $twig
      * @return static
      */
-    public static function create(TwigView $twigView, \Twig_Environment $twig)
+    public static function create(TwigView $twigView, Environment $twig)
```

Comment:
Could add return type hint `: ConstructEvent`

Reply:
Did this everywhere aside for mixed cases.